rest/route/project_copy.go

```diff
@@ -74,6 +74,7 @@ func (p *projectCopyHandler) Run(ctx context.Context) gimlet.Responder {
 	projectToCopy.Enabled = utility.FalsePtr()
 	projectToCopy.PRTestingEnabled = nil
 	projectToCopy.CommitQueue.Enabled = nil
+	adminsToAdd := projectToCopy.Admins
```

Comment:
We don't need to save this variable, since projectToCopy.Admins isn't modified you can just reference it directly at 98